doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1615,6 +1615,10 @@ annotation: views are interpreted opening the ``ssripat`` scope.
   a new constant, fact, or defined constant :token:`ident`, respectively.
   Note that defined constants cannot be introduced when δ-expansion is
   required to expose the top variable or assumption.
+  An empty occurrence switch ``{}`` has the effect of clearing
```

Comment:
For more clarity, what about changing the line above:

```
  :token:`ident`
```

into

```
  :n:`{? %{%} } @ident`
```

Reply:
Hum, let's see how #9341 goes, since it generalizes it to non-empty clears...